.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: ruby
 rvm:
-  - 2.2.2
-  - 2.1.3
+  - 2.4.1
```

Comment:
no need to test with 2.4.1 and 2.4.2, you should only test multiple major versions

Reply:
it's not true, btw, rails 5.1 requires ruby >= 2.2.2 - see https://rubygems.org/gems/rails/versions/5.1.4. pls make sure it works with 2.3.x and 2.4.x, update the min required version in the gemspec to `>=2.3.0`